### PR TITLE
Update supported-data-platforms.md

### PR DESCRIPTION
**Pull request by alexanderrw-lib**
There was a mistyped section on the page supported data platforms page. I've added what I _assume_ is the intended sentence, but open to alternative copy (or just removing point 6 entirely).

## What are you changing in this pull request and why?
Minor copy changes to the 'supported data platforms' page.

## Checklist

- [ ✅  ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ❓ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

Adding or removing pages (delete if not applicable):
None of these were applicable - minor copy changes only.

### CODE REVIEW

**Comment by alexanderrw-lib** [comment]
I wasn't sure if i needed to [version this](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) considering how small the change was?

File: website/docs/docs/supported-data-platforms.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ There are two types of adapters available today:
 1. Does it work?
 2. Does anyone "own" the code, or is anyone liable for ensuring it works?
 3. Do bugs get fixed quickly?
-4. Does it stay up-to-date with new Core features?
+4. Does it stay up-to-date with new dbt Core features?
 5. Is the usage substantial enough to self-sustain?
-pendency on this library?
+6. Do you already have a dependency on this library?
```
Comment (dataders) [inline website/docs/docs/supported-data-platforms.md]: I have no memory of what this question might have been, but perhaps this one is helpful @alexanderrw-lib?

```suggestion
6. Do other well-known well-adopted projects already depend on this library?
```

Comment (alexanderrw-lib) [reply website/docs/docs/supported-data-platforms.md]: Significantly better - thank you @dataders !

**Review by dataders — APPROVED** [review]
thanks for your help @alexanderrw-lib!

File: website/docs/docs/supported-data-platforms.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ There are two types of adapters available today:
 1. Does it work?
 2. Does anyone "own" the code, or is anyone liable for ensuring it works?
 3. Do bugs get fixed quickly?
-4. Does it stay up-to-date with new Core features?
+4. Does it stay up-to-date with new dbt Core features?
 5. Is the usage substantial enough to self-sustain?
-pendency on this library?
+6. Do other well-known well-adopted projects already depend on this library?
```
Comment (mirnawong1) [inline website/docs/docs/supported-data-platforms.md]: simplifying a bit bc there were a lot of 'well' and i think the intent is just 'popular' or known' projects. pls let me know if this is't the right tweak and happy to follow up @dataders !
```suggestion
6. Do other known projects depend on this library?
```

**Comment by mirnawong1** [comment]
merging this for you @alexanderrw-lib and @dataders , thanks again for opening this up and collabing on this ✨

**Pull request merged**